packages/sandbox/src/app/sandbox/docs/navigation-bar-trail.doc.tsx

```diff
@@ -0,0 +1,69 @@
+import { NavigationBarTrail, IconsName } from '@getluko/streamline';
+
+import { DocList } from '../components/DocList';
+
+const left = {
```

Comment:
`const left:  ButtonIconProps = {` to prevent having the `as IconsName` and infer type

Reply:
I don't why, but I can't import ButtonIconProps from @getluko/streamline